packages/cli/src/commands/local.ts

```diff
@@ -50,7 +50,7 @@ export default class LocalCommand extends Command {
     'skip-wait-for-ipfs': Flags.boolean({
       summary: "Don't wait for IPFS to be up at localhost:15001",
     }),
-    'skip-wait-for-etherium': Flags.boolean({
```

Comment:
great catch but we would need to do a breaking change, we should deprecate this old arg, show a warning to user and drop this in a next major

Reply:
@saihaj Thank you for your review. Unfortunately, I'm not a developer, and I wouldn't want to inadvertently cause any issues, so it would be greatly appreciated if you could handle the deprecation and warning tasks.